tests/Integration/Adapter/SqlManager/SqlQueryValidatorTest.php

```diff
@@ -59,6 +59,22 @@ public function dataProviderValidate(): iterable
             . 'GROUP BY c.id_customer;',
             [],
         ];
+        yield [
+            'SELECT SQL_CALC_FOUND_ROWS b.*, a.* '
+            . 'FROM `ps_cart_rule` a '
+            . 'LEFT JOIN `ps_cart_rule_lang` b ON (b.`id_cart_rule` = a.`id_cart_rule` AND b.`id_lang` = 1) '
+            . 'WHERE 1 '
+            . 'ORDER BY a.id_cart_rule DESC',
+            [],
+        ];
```

Comment:
```suggestion
        yield [
            'SELECT SQL_CALC_FOUND_ROWS b.*, a.* '
            . 'FROM `ps_cart_rule` a '
            . 'LEFT JOIN `ps_cart_rule_lang` b ON (b.`id_cart_rule` = a.`id_cart_rule` AND b.`id_lang` = 1) '
            . 'WHERE 1 '
            . 'ORDER BY a.id_cart_rule DESC',
            [],
        ];
        yield [
            'SELECT SQL_CALC_FOUND_ROWS b.*, a.* '
            . 'FROM `ps_cart_rule` a '
            . 'LEFT JOIN `ps_cart_rule_lang` b ON (b.`id_cart_rule` = a.`id_cart_rule` AND b.`id_lang` = a.`id_lang`) '
            . 'WHERE 1 '
            . 'ORDER BY a.id_cart_rule DESC',
            [],
        ];
```

Reply:
I experiment differents behaviors when we have something different than 1. To be sure everything is ok

Reply:
Test added.